management_interface/management_interface/admin.py

```diff
@@ -30,6 +34,47 @@ def save_model(self, request, obj, form, change):
         obj = set_obj_created_updated(request, obj, form)
         super().save_model(request, obj, form, change)
 
+    def delete_queryset(self, request, queryset: Iterable[CareRecipient]):
+        for care_recipient in queryset:
+            try:
+                ManagementAPIClient().delete_subscription(
```

Comment:
I think we should put this line over in `CareRecipient.delete()`, keeping the rest of the code as-is.

My reasoning is:

1. It prevents single deletes being made by accident elsewhere in the code without calling the API
2. Ties the delete API call behaviour to the model (in the same way as the create call)
3. You can (and will need to) keep the rest of the code here exactly the same (because `delete()` is not called by default when processing a QuerySet)